docs: improve the prerequisites of RWX volume (#672)
harvester/docs: docs/rancher/csi-driver.md

  ![](/img/v1.4/rancher/harvester-csi-driver-version.png)

> - Each nodes in the guest cluster needs to have the NFS client installed.

Reviewer: ```suggestion
- Each node in the guest cluster must have the NFS client installed.
```

> - Each node in the guest cluster must have the NFS client installed.
> 
>   The command to install the NFS client is as follows:
> 
>   - For Debian and Ubuntu:
>   ```
>   apt-get install -y nfs-common
>   ```
> 
>   - For CentOS and RHEL:
>   ```
>   yum install -y nfs-utils
>   ```
> 
>   - For SUSE/OpenSUSE:
>   ```
>   zypper install -y nfs-client
>   ```

Reviewer 2: ```suggestion
- The NFS client is installed on each node in the guest cluster.

  Run any of the following commands to install the NFS client.

  - Debian and Ubuntu: `apt-get install -y nfs-common`
  
  - CentOS and RHEL: `yum install -y nfs-utils`

  - SUSE and OpenSUSE: `zypper install -y nfs-client`
```

Author: Thanks for the suggestion!  Updated!
